src/Entity/Token.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Pucene\Bundle\DoctrineBundle\Entity;
+
+/**
+ * Token entity.
+ */
+class Token
```

Comment:
as said we need todo a benchmark if its faster to have an additional entity or store the string here directly.